src/adapters/Cornerstone3D/MeasurementReport.js

```diff
@@ -77,7 +77,79 @@ function getMeasurementGroup(
 export default class MeasurementReport {
     constructor() {}
 
-    static getSetupMeasurementData(MeasurementGroup) {
+    static getCornerstoneLabelFromDefaultState(defaultState) {
+        const { findingSites = [], finding } = defaultState;
+
+        let freeTextLabel = findingSites.find(
+            fs => fs.CodeValue === "CORNERSTONEFREETEXT"
```

Comment:
can we use CodeScheme from here (src/adapters/Cornerstone3D/CodingScheme.js) for the `CORNERSTONEFREETEXT`

Reply:
good catch